scenarios/fuel_test/contracts/interaction-tools/src/main.rs

```diff
@@ -65,16 +69,16 @@ async fn main() -> Result<()> {
     println!("Deposited 100 coins to LP in tx: 0x{}", r.tx_id.unwrap());
     let lp_asset_id = contract_id.asset_id(&Bits256::zeroed());
     let lp_token_balance = wallet.get_asset_balance(&lp_asset_id).await?;
-    let call_params = CallParameters::default()
+    let _call_params = CallParameters::default()
         .with_amount(lp_token_balance)
         .with_asset_id(lp_asset_id);
-    let r = contract_methods
-        .withdraw(wallet.address().into())
-        .call_params(call_params)?
-        .with_variable_output_policy(VariableOutputPolicy::Exactly(1))
-        .with_contract_ids(&[greeter_contract_id.into()])
-        .call()
-        .await?;
+    // let r = contract_methods
+    //     .withdraw(wallet.address().into())
+    //     .call_params(call_params)?
+    //     .with_variable_output_policy(VariableOutputPolicy::Exactly(1))
+    //     .with_contract_ids(&[greeter_contract_id.into()])
+    //     .call()
+    //     .await?;
```

Comment:
Looks like the println! below will get the wrong intended value for the variable `r` since it was being shadowed